modules/rostests/winetests/cabinet/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 
 add_definitions(-DUSE_WINE_TODOS)
+remove_definitions(-D__ROS_LONG64__)
```

Comment:
Question for helping my understanding: what does that added line fix or improve? Why is it neccessary?